src/server/views/dashboard/lists.njk

```diff
@@ -31,7 +31,7 @@
         {% for list in lists %}
           <tr class="govuk-table__row">
             <td scope="row" class="govuk-table__header">
-                <a href="{{ listIndexUrl }}" class="govuk-link">{{ _.startCase(list.type) }}</a>
+                <a href="{{ dashboardRoutes.listsItems.replace(':listId', list.id) }}" class="govuk-link">{{ _.startCase(list.type) }}</a>
```

Comment:
For some reason when I do that, the url doesn't seem to exist? Not sure if it's something I need to add to the controller.

<img width="1087" alt="Screenshot 2023-01-03 at 13 38 25" src="https://user-images.githubusercontent.com/1377253/210368173-b79f6e86-3c33-43df-a84c-3fcaba0977bb.png">
<img width="1048" alt="Screenshot 2023-01-03 at 13 38 19" src="https://user-images.githubusercontent.com/1377253/210368201-11b938d9-3c20-4078-b34b-ab965182e578.png">

Reply:
I guess because it's using listRouter.get("/", listsController); as a route and because no listId is being passed in as a param it doesn't produce the data in locals

Reply:
ah sorry, I see, it's just iterating through an array of list rather than using req.param. ignore 🙂